server/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const mongoose = require("mongoose");
+const routes = require("./routes/index");
+const helpers = require("./helpers");
+const path = require('path');
+
+const app = express();
+
+app.set("views", path.join(__dirname, "views"));
+app.set("view engine", "pug");
+// app.set("vendors", path.join(__dirname, "../vendors"))
```

Comment:
Can this comment be removed now? :) 
It's best not to leave commented out code in your work as other developers who read it don't know why it's there.

Reply:
Thank you for your review. 
Bootstrap: I've been struggling with bootstrap working with pug - it was the only way I could set it up. We can look into it on Wednesday. I had a jade-boostrap package installed, and now I tried installing just bootstrap npm - column styling still is lost with it. I had the same issue with the font-awesome, which I resolved the same way as bootstrap. I wonder if I've missed a "require"?
Commented out code: removed.
Thank you.

Reply:
No problem, let's talk about it on Monday. In the meantime, you can still git ignore the bootstrap folder which would at least make the rest of the PR easier to review :) Message me if you need any help with adding it to your git ignore file.

Reply:
Heya, the node_module, bootstrap, fontawesome folders were removed from the repo (removed-but-not-removed, added, double added, finally removed and history cleaned). I've used rebase and git gui to make the changes and pushed them into the git.
Awaiting your comments.